emission/integrationTests/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3"
+services:
+  web-server:
+    build: 
+      context: .
+    depends_on:
+      - db
+      - nominatim
+    environment:
+      - DB_HOST=db
+      - WEB_SERVER_HOST=0.0.0.0
+      - NOMINATIM_QUERY_URL=http://rhodeisland-nominatim:8080
+      - GFBK_KEY=$GFBK_KEY
+    volumes:
+      # specify the host directory where the source code should live
+      # If this is ~/e-mission-server-docker, then you can edit the files at
+      # ~/e-mission-server-docker/src/e-mission-server/emission/...
+      # - CHANGEME:/src/
+      - ../..:/src/e-mission-server
+    networks:
+       - emission
+       - nominatim
+
+  db:
+    image: mongo:4.4.0
+    ports:
+      - "1234:1234"
```

Comment:
why are we exporting a port here?

Reply:
I was running into issues with not specifying the ports. This was based on what I found on networking within GitHub actions. I'll try to run without it and see if it is necessary.

Reply:
This port was not necessary for the test to succeed. However, removing the port for the container caused all of the tests that call the container to give ` urllib.error.HTTPError: HTTP Error 404: Not Found`

Reply:
that makes no sense. By "tests that call the container", which container do you mean? This is a 404 error, so it must be a container that is accessed via HTTP, so the nominatim container? But the nominatim container doesn't use mongodb - it can run completely independently. So I don't see how any such error could be related. Please clarify/add additional logs.

Reply:
I removed the mongo container's port and the test worked just fine, since none of the other integration tests are currently run during my workflow. 

I also tried removing the port for the nominatim container. The rhodeisland-nominatim container is called by multiple tests within `TestNominatim.py`, each of which could not call the container without a port exposed. The point I was getting at is that the nominatim container still requires a port to be called by the tests in web-server.

General: you mentioned that we [want the mongo container present](https://github.com/e-mission/e-mission-docs/issues/937#issuecomment-1666231008), presumably for when all of the integration tests in the `integrationTests` directory are run together (assuming this is what you want). We definitely don't need the mongo port now, but once the [other tests are incorporated ](https://github.com/e-mission/e-mission-docs/issues/937#issuecomment-1686953729) the port might have to return to be called by web-server.